public/app/plugins/datasource/cloudwatch/plugin.json

```diff
@@ -5,6 +5,7 @@
   "category": "cloud",
   "metrics": true,
   "logs": true,
+  "live": true,
```

Comment:
Is this necessary now?  I think we will need something like it eventually, but lets be careful about how we support it.  Ideally it can actually be driven by the existence of interfaces and backend support.

Reply:
Not necessary, think this is just a holdover from an earlier version of Live